lte/gateway/c/session_manager/PipelinedClient.cpp

```diff
@@ -91,13 +128,52 @@ magma::ActivateFlowsRequest create_activate_req(
   if (ambr) {
     req.mutable_apn_ambr()->CopyFrom(*ambr);
   }
-  auto mut_versioned_rules = req.mutable_policies();
+  return req;
+}
+
+/**
+ * @brief Create a map of Teids -> ActivateFlowsRequest
+ * If to_process is empty, create one default_teids -> ActivateFlowsRequest with
+ * no rules. For each to_process item, create item.teid -> ActivateFlowsRequest
+ * with item.rules
+ * @param imsi
+ * @param ip_addr
+ * @param ipv6_addr
+ * @param default_teids
+ * @param msisdn
+ * @param ambr
+ * @param to_process
+ * @param origin_type
+ * @return magma::ActivateReqByTeids
+ */
+magma::ActivateReqByTeids make_activate_req_by_teid(
+    const std::string& imsi, const std::string& ip_addr,
+    const std::string& ipv6_addr, const magma::Teids default_teids,
+    const std::string& msisdn,
+    const optional<magma::AggregatedMaximumBitrate>& ambr,
+    const magma::RulesToProcess& to_process,
+    const magma::RequestOriginType_OriginType origin_type) {
+  magma::ActivateReqByTeids activate_req_by_teids;
+  if (to_process.empty()) {
+    // Send an empty request with the default teid
+    activate_req_by_teids[default_teids] = make_activate_req(
+        imsi, ip_addr, ipv6_addr, default_teids, msisdn, ambr, origin_type);
+    return activate_req_by_teids;
+  }
```

Comment:
if we dont have rules to process, why do we want to activate?

Reply:
I think we rarely hit this case, but even if there are no policies we need to install the drop all rule. So for session creation/activation, even in the rare case we have no policies, we need to send a request.

Reply:
cc @koolzz for confirmation :D

Reply:
yeah thats correct :) we implicitly drop all ue traffic in that case